src/main/java/lotto/domain/Lotties.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lotties {
+    private final List<Lotto> lotties;
+
+    public Lotties(List<Lotto> lotties) {
+        this.lotties = new ArrayList<>(lotties);
+    }
+
+    public int getLottiesSize() {
```

Comment:
```suggestion
    public int size() {
```
엇 이전 코멘트는 요렇게 바꿔보시는걸 권유드렸던 거였습니다ㅎㅎㅎ
일급"컬렉션"이니 사실상 컬렉션처럼 사용하는 것이 더 읽기 좋을 것 같아요~